templates/index.html

```diff
@@ -1,5 +1,11 @@
-{% extends "base_index.html" %} {% block page_class %}homepage{% endblock %} {% block page-title %}| Home{% endblock %} {% block head-extra %}
-{% endblock %} {% block content %}
+{% extends "base_index.html" %}
+
+{% block page_class %}homepage{% endblock %}
+{% block page-title %}| Home{% endblock %}
```

Comment:
I wonder if this would be better being a short description of the product instead? Otherwise it will just say `MAAS | Home` on googles search results.